lms/static/sass/base/_variables.scss

```diff
@@ -82,6 +82,27 @@ $gray-d2: shade($gray,40%); // #4c4c4c
 $gray-d3: shade($gray,60%); // #323232
 $gray-d4: shade($gray,80%); // #191919
 
+$cms-blue: rgb(0, 159, 230); // NOTE - eventually, this set of blues should be synced with CMS.  
```

Comment:
While it sucks to have all our blues start with m- I don't think we should put in a whole new set. Can you use the other blue variables for now? Feel free to make a followup cleanup story for our backlog.